step-templates/f5-ltm-rest-enable-disable-member.json

```diff
@@ -0,0 +1,114 @@
+  },
+  "Parameters": [
+    {
+      "Id": "5ed914ec-ff22-436d-ac37-1ccc5216372e",
+      "Name": "WaitForConnections",
+      "Label": "Wait on Connections?",
+      "HelpText": "If checked, the deployment won't continue until current connections on the node = 0",
```

Comment:
It could be helpful to note that the connection count can be configured. What do you think about reordering the parameters so that this and connection count are next to each other?

Reply:
moved connection count variable next to WaitForConnections in commit c0cc56b1079c030a757a0d14121cea2f15bd4fd7